api/src/main/java/org/apache/cloudstack/api/command/admin/offering/UpdateServiceOfferingCmd.java

```diff
@@ -71,6 +73,20 @@
             since = "4.13")
     private String zoneIds;
 
+    @Parameter(name = ApiConstants.TAGS,
```

Comment:
STORAGE_TAGS might be better

Reply:
Makes sense to create `ApiConstants.STORAGE_TAGS` to differ from generic tags, similar to what we have at the `ApiConstants.HOST_TAGS`.